src/app/issue-management/issue-management-detail/issue-management-detail.component.ts

```diff
@@ -174,6 +189,7 @@ export class IssueManagementDetailComponent implements OnInit, AfterViewInit {
     this.issueManagementService.updateIssue(this.issue).subscribe(result => {
       this.issue = result;
       this.disabled = true;
+      this.router.navigate(['./issue/detail', this.issue.name]);
```

Comment:
Was this navigation missing for the candidates on update?
To suppress double updates you cuold store the last parameter values in the component and check if tehy have changed in the route subscription. Only performing an update if they have changed. Then if a navigation does not need to perform an update internally change the values in the component to the new expected ones and then navigate to update the URL to match the internal state.